docs/metrics.md

```diff
@@ -0,0 +1,166 @@
+## Prometheus Metrics
+
+### List of Metrics Exported By Prometheus
```

Comment:
As noted at https://github.com/prometheus/prometheus/issues/12397#issuecomment-2240879670, this should be caveated as a "Partial list" or similar.

Reply:
Hi @bboreham, you welcome!, I have caveated it as a Partial list with a new commit.